src/jdk.compiler/share/classes/com/sun/tools/javac/launcher/SourceLauncher.java

```diff
@@ -261,12 +261,15 @@ private Class<?> execute(MemoryContext context, String[] mainArgs)
             throw new Fault(Errors.CantAccessMainMethod(mainClassName));
         } catch (InvocationTargetException exception) {
             // remove stack frames for source launcher
+            StackTraceElement[] invocationElements = exception.getStackTrace();
+            if (invocationElements == null) throw exception;
```

Comment:
`invocationElements` can’t be `null`, as the `InvocationTargetException` is thrown by the reflection implementation:
https://github.com/openjdk/jdk/blob/3fd89be6d1a51b6fc99f4c0b5daba7a4bd64a08e/src/java.base/share/classes/jdk/internal/reflect/DirectMethodHandleAccessor.java#L103-L120